<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>microbiomeDataSets • microbiomeDataSets</title>
<!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script><!-- Bootstrap --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/css/bootstrap.min.css" integrity="sha256-bZLfwXAP04zRMK2BjiO8iu9pf4FbLqX6zitd+tIvLhE=" crossorigin="anonymous">
<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="../bootstrap-toc.css">
<script src="../bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script><meta property="og:title" content="microbiomeDataSets">
<meta property="og:description" content="microbiomeDataSets">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body data-spy="scroll" data-target="#toc">
    <div class="container template-article">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">microbiomeDataSets</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="Released version">1.1.2</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li>
  <a href="../index.html">
    <span class="fas fa-home fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="../articles/microbiomeDataSets.html">Get started</a>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right"></ul>
</div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->

      

      </header><script src="microbiomeDataSets_files/header-attrs-2.9/header-attrs.js"></script><div class="row">
  <div class="col-md-9 contents">
    <div class="page-header toc-ignore">
      <h1 data-toc-skip>microbiomeDataSets</h1>
            
      
      
      <div class="hidden name"><code>microbiomeDataSets.Rmd</code></div>

    </div>

    
    
<div class="sourceCode" id="cb1"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span><span class="op">(</span><span class="va">microbiomeDataSets</span><span class="op">)</span>
<span class="co">#&gt; Loading required package: SummarizedExperiment</span>
<span class="co">#&gt; Loading required package: MatrixGenerics</span>
<span class="co">#&gt; Loading required package: matrixStats</span>
<span class="co">#&gt; </span>
<span class="co">#&gt; Attaching package: 'MatrixGenerics'</span>
<span class="co">#&gt; The following objects are masked from 'package:matrixStats':</span>
<span class="co">#&gt; </span>
<span class="co">#&gt;     colAlls, colAnyNAs, colAnys, colAvgsPerRowSet, colCollapse,</span>
<span class="co">#&gt;     colCounts, colCummaxs, colCummins, colCumprods, colCumsums,</span>
<span class="co">#&gt;     colDiffs, colIQRDiffs, colIQRs, colLogSumExps, colMadDiffs,</span>
<span class="co">#&gt;     colMads, colMaxs, colMeans2, colMedians, colMins, colOrderStats,</span>
<span class="co">#&gt;     colProds, colQuantiles, colRanges, colRanks, colSdDiffs, colSds,</span>
<span class="co">#&gt;     colSums2, colTabulates, colVarDiffs, colVars, colWeightedMads,</span>
<span class="co">#&gt;     colWeightedMeans, colWeightedMedians, colWeightedSds,</span>
<span class="co">#&gt;     colWeightedVars, rowAlls, rowAnyNAs, rowAnys, rowAvgsPerColSet,</span>
<span class="co">#&gt;     rowCollapse, rowCounts, rowCummaxs, rowCummins, rowCumprods,</span>
<span class="co">#&gt;     rowCumsums, rowDiffs, rowIQRDiffs, rowIQRs, rowLogSumExps,</span>
<span class="co">#&gt;     rowMadDiffs, rowMads, rowMaxs, rowMeans2, rowMedians, rowMins,</span>
<span class="co">#&gt;     rowOrderStats, rowProds, rowQuantiles, rowRanges, rowRanks,</span>
<span class="co">#&gt;     rowSdDiffs, rowSds, rowSums2, rowTabulates, rowVarDiffs, rowVars,</span>
<span class="co">#&gt;     rowWeightedMads, rowWeightedMeans, rowWeightedMedians,</span>
<span class="co">#&gt;     rowWeightedSds, rowWeightedVars</span>
<span class="co">#&gt; Loading required package: GenomicRanges</span>
<span class="co">#&gt; Loading required package: stats4</span>
<span class="co">#&gt; Loading required package: BiocGenerics</span>
<span class="co">#&gt; </span>
<span class="co">#&gt; Attaching package: 'BiocGenerics'</span>
<span class="co">#&gt; The following objects are masked from 'package:stats':</span>
<span class="co">#&gt; </span>
<span class="co">#&gt;     IQR, mad, sd, var, xtabs</span>
<span class="co">#&gt; The following objects are masked from 'package:base':</span>
<span class="co">#&gt; </span>
<span class="co">#&gt;     anyDuplicated, append, as.data.frame, basename, cbind, colnames,</span>
<span class="co">#&gt;     dirname, do.call, duplicated, eval, evalq, Filter, Find, get, grep,</span>
<span class="co">#&gt;     grepl, intersect, is.unsorted, lapply, Map, mapply, match, mget,</span>
<span class="co">#&gt;     order, paste, pmax, pmax.int, pmin, pmin.int, Position, rank,</span>
<span class="co">#&gt;     rbind, Reduce, rownames, sapply, setdiff, sort, table, tapply,</span>
<span class="co">#&gt;     union, unique, unsplit, which.max, which.min</span>
<span class="co">#&gt; Loading required package: S4Vectors</span>
<span class="co">#&gt; </span>
<span class="co">#&gt; Attaching package: 'S4Vectors'</span>
<span class="co">#&gt; The following objects are masked from 'package:base':</span>
<span class="co">#&gt; </span>
<span class="co">#&gt;     expand.grid, I, unname</span>
<span class="co">#&gt; Loading required package: IRanges</span>
<span class="co">#&gt; Loading required package: GenomeInfoDb</span>
<span class="co">#&gt; Loading required package: Biobase</span>
<span class="co">#&gt; Welcome to Bioconductor</span>
<span class="co">#&gt; </span>
<span class="co">#&gt;     Vignettes contain introductory material; view with</span>
<span class="co">#&gt;     'browseVignettes()'. To cite Bioconductor, see</span>
<span class="co">#&gt;     'citation("Biobase")', and for packages 'citation("pkgname")'.</span>
<span class="co">#&gt; </span>
<span class="co">#&gt; Attaching package: 'Biobase'</span>
<span class="co">#&gt; The following object is masked from 'package:MatrixGenerics':</span>
<span class="co">#&gt; </span>
<span class="co">#&gt;     rowMedians</span>
<span class="co">#&gt; The following objects are masked from 'package:matrixStats':</span>
<span class="co">#&gt; </span>
<span class="co">#&gt;     anyMissing, rowMedians</span>
<span class="co">#&gt; Loading required package: TreeSummarizedExperiment</span>
<span class="co">#&gt; Loading required package: SingleCellExperiment</span>
<span class="co">#&gt; Loading required package: Biostrings</span>
<span class="co">#&gt; Loading required package: XVector</span>
<span class="co">#&gt; </span>
<span class="co">#&gt; Attaching package: 'Biostrings'</span>
<span class="co">#&gt; The following object is masked from 'package:base':</span>
<span class="co">#&gt; </span>
<span class="co">#&gt;     strsplit</span>
<span class="co">#&gt; Loading required package: MultiAssayExperiment</span></code></pre></div>
<div id="microbiome-example-data-sets" class="section level1">
<h1 class="hasAnchor">
<a href="#microbiome-example-data-sets" class="anchor"></a>Microbiome example data sets</h1>
<p>The data sets are primarily named by the first author of the associated publication, together with a descriptive suffix. Aliases are provided for some of the data sets.</p>
<p>A table of the available data sets is available through the <code>availableDataSets</code> function.</p>
<div class="sourceCode" id="cb2"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="fu"><a href="../reference/availableDataSets.html">availableDataSets</a></span><span class="op">(</span><span class="op">)</span>
<span class="co">#&gt;             Dataset</span>
<span class="co">#&gt; 1    HintikkaXOData</span>
<span class="co">#&gt; 2       LahtiMLData</span>
<span class="co">#&gt; 3        LahtiMData</span>
<span class="co">#&gt; 4       LahtiWAData</span>
<span class="co">#&gt; 5      OKeefeDSData</span>
<span class="co">#&gt; 6 SilvermanAGutData</span>
<span class="co">#&gt; 7        SongQAData</span>
<span class="co">#&gt; 8   SprockettTHData</span></code></pre></div>
<p>The data is usually loaded as a <em><a href="https://bioconductor.org/packages/3.14/TreeSummarizedExperiment">TreeSummarizedExperiment</a></em> or a <em><a href="https://bioconductor.org/packages/3.14/MultiAssayExperiment">MultiAssayExperiment</a></em>, if microbiome associated data was used in the respective study. For more information on how to use these objects, please refer to the vignettes of the packages.</p>
<div id="hit-chip-data" class="section level2">
<h2 class="hasAnchor">
<a href="#hit-chip-data" class="anchor"></a>HIT Chip Data</h2>
<p>The following data sets are based on the Human Intestinal Tract (HIT)Chip phylogenetic 16S microarray <a href="https://doi.org/10.1111/j.1462-2920.2009.01900.x">(Rajilić-Stojanović <em>et al.</em> 2009)</a>. This profiling technology differs from the more widely used 16S rRNA amplicon sequencing.</p>
<p>Since the data is also available in phyloseq format through the microbiome R package and are referenced usually via aliases, they are described in more detail.</p>
<div id="intestinal-microbiota-profiling-of-1006-western-adults" class="section level3">
<h3 class="hasAnchor">
<a href="#intestinal-microbiota-profiling-of-1006-western-adults" class="anchor"></a>Intestinal microbiota profiling of 1006 Western adults</h3>
<p>This data set from <a href="http://www.nature.com/ncomms/2014/140708/ncomms5344/full/ncomms5344.html">Lahti et al. Nat. Comm. 5:4344, 2014</a> comes with 130 genus-like taxonomic groups across 1006 western adults with no reported health complications. Some subjects have also short time series.</p>
<div class="sourceCode" id="cb3"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="fu"><a href="../reference/LahtiWAData.html">LahtiWAData</a></span><span class="op">(</span><span class="op">)</span>
<span class="co">#&gt; class: TreeSummarizedExperiment </span>
<span class="co">#&gt; dim: 130 1151 </span>
<span class="co">#&gt; metadata(0):</span>
<span class="co">#&gt; assays(1): counts</span>
<span class="co">#&gt; rownames(130): Actinomycetaceae Aerococcus ... Xanthomonadaceae</span>
<span class="co">#&gt;   Yersinia et rel.</span>
<span class="co">#&gt; rowData names(3): Phylum Family Genus</span>
<span class="co">#&gt; colnames(1151): Sample-1 Sample-2 ... Sample-1171 Sample-1172</span>
<span class="co">#&gt; colData names(10): age sex ... time sample</span>
<span class="co">#&gt; reducedDimNames(0):</span>
<span class="co">#&gt; mainExpName: NULL</span>
<span class="co">#&gt; altExpNames(0):</span>
<span class="co">#&gt; rowLinks: NULL</span>
<span class="co">#&gt; rowTree: NULL</span>
<span class="co">#&gt; colLinks: NULL</span>
<span class="co">#&gt; colTree: NULL</span>

<span class="co"># Alias</span>
<span class="co"># atlas1006()</span></code></pre></div>
</div>
<div id="diet-swap-between-rural-and-western-populations" class="section level3">
<h3 class="hasAnchor">
<a href="#diet-swap-between-rural-and-western-populations" class="anchor"></a>Diet swap between Rural and Western populations</h3>
<p>A two-week diet swap study between western (USA) and traditional (rural Africa) diets, reported in <a href="http://dx.doi.org/10.1038/ncomms7342">O’Keefe et al. Nat. Comm. 6:6342, 2015</a>.</p>
<div class="sourceCode" id="cb4"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="fu"><a href="../reference/OKeefeDSData.html">OKeefeDSData</a></span><span class="op">(</span><span class="op">)</span>
<span class="co">#&gt; snapshotDate(): 2021-08-04</span>
<span class="co">#&gt; see ?microbiomeDataSets and browseVignettes('microbiomeDataSets') for documentation</span>
<span class="co">#&gt; loading from cache</span>
<span class="co">#&gt; see ?microbiomeDataSets and browseVignettes('microbiomeDataSets') for documentation</span>
<span class="co">#&gt; loading from cache</span>
<span class="co">#&gt; see ?microbiomeDataSets and browseVignettes('microbiomeDataSets') for documentation</span>
<span class="co">#&gt; loading from cache</span>
<span class="co">#&gt; class: TreeSummarizedExperiment </span>
<span class="co">#&gt; dim: 130 222 </span>
<span class="co">#&gt; metadata(0):</span>
<span class="co">#&gt; assays(1): counts</span>
<span class="co">#&gt; rownames(130): Actinomycetaceae Aerococcus ... Xanthomonadaceae</span>
<span class="co">#&gt;   Yersinia et rel.</span>
<span class="co">#&gt; rowData names(3): Phylum Family Genus</span>
<span class="co">#&gt; colnames(222): Sample-1 Sample-2 ... Sample-221 Sample-222</span>
<span class="co">#&gt; colData names(8): subject sex ... timepoint.within.group bmi_group</span>
<span class="co">#&gt; reducedDimNames(0):</span>
<span class="co">#&gt; mainExpName: NULL</span>
<span class="co">#&gt; altExpNames(0):</span>
<span class="co">#&gt; rowLinks: NULL</span>
<span class="co">#&gt; rowTree: NULL</span>
<span class="co">#&gt; colLinks: NULL</span>
<span class="co">#&gt; colTree: NULL</span>

<span class="co"># Alias</span>
<span class="co"># dietswap()</span></code></pre></div>
</div>
<div id="intestinal-microbiota-versus-blood-metabolites" class="section level3">
<h3 class="hasAnchor">
<a href="#intestinal-microbiota-versus-blood-metabolites" class="anchor"></a>Intestinal microbiota versus blood metabolites</h3>
<p>Data set from <a href="https://doi.org/10.7717/peerj.32">Lahti et al. PeerJ 1:e32, 2013</a> characterizes associations between human intestinal microbiota and blood serum lipids. Note that this data set contains an additional assay of lipid species, and is therefore provided as <code>MultiAssayExperiment</code> object.</p>
<div class="sourceCode" id="cb5"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="fu"><a href="../reference/LahtiMLData.html">LahtiMLData</a></span><span class="op">(</span><span class="op">)</span>
<span class="co">#&gt; snapshotDate(): 2021-08-04</span>
<span class="co">#&gt; see ?microbiomeDataSets and browseVignettes('microbiomeDataSets') for documentation</span>
<span class="co">#&gt; loading from cache</span>
<span class="co">#&gt; see ?microbiomeDataSets and browseVignettes('microbiomeDataSets') for documentation</span>
<span class="co">#&gt; loading from cache</span>
<span class="co">#&gt; see ?microbiomeDataSets and browseVignettes('microbiomeDataSets') for documentation</span>
<span class="co">#&gt; loading from cache</span>
<span class="co">#&gt; see ?microbiomeDataSets and browseVignettes('microbiomeDataSets') for documentation</span>
<span class="co">#&gt; loading from cache</span>
<span class="co">#&gt; A MultiAssayExperiment object of 2 listed</span>
<span class="co">#&gt;  experiments with user-defined names and respective classes.</span>
<span class="co">#&gt;  Containing an ExperimentList class object of length 2:</span>
<span class="co">#&gt;  [1] microbiome: TreeSummarizedExperiment with 130 rows and 44 columns</span>
<span class="co">#&gt;  [2] lipids: SummarizedExperiment with 389 rows and 44 columns</span>
<span class="co">#&gt; Functionality:</span>
<span class="co">#&gt;  experiments() - obtain the ExperimentList instance</span>
<span class="co">#&gt;  colData() - the primary/phenotype DataFrame</span>
<span class="co">#&gt;  sampleMap() - the sample coordination DataFrame</span>
<span class="co">#&gt;  `$`, `[`, `[[` - extract colData columns, subset, or experiment</span>
<span class="co">#&gt;  *Format() - convert into a long or wide DataFrame</span>
<span class="co">#&gt;  assays() - convert ExperimentList to a SimpleList of matrices</span>
<span class="co">#&gt;  exportClass() - save all data to files</span>

<span class="co"># Alias</span>
<span class="co"># peerj32()</span></code></pre></div>
</div>
<div id="version-info" class="section level3">
<h3 class="hasAnchor">
<a href="#version-info" class="anchor"></a>Version info</h3>
<div class="sourceCode" id="cb6"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="fu"><a href="https://rdrr.io/r/utils/sessionInfo.html">sessionInfo</a></span><span class="op">(</span><span class="op">)</span>
<span class="co">#&gt; R version 4.1.0 (2021-05-18)</span>
<span class="co">#&gt; Platform: x86_64-pc-linux-gnu (64-bit)</span>
<span class="co">#&gt; Running under: Ubuntu 20.04.2 LTS</span>
<span class="co">#&gt; </span>
<span class="co">#&gt; Matrix products: default</span>
<span class="co">#&gt; BLAS/LAPACK: /usr/lib/x86_64-linux-gnu/openblas-pthread/libopenblasp-r0.3.8.so</span>
<span class="co">#&gt; </span>
<span class="co">#&gt; locale:</span>
<span class="co">#&gt;  [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              </span>
<span class="co">#&gt;  [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    </span>
<span class="co">#&gt;  [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=C             </span>
<span class="co">#&gt;  [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 </span>
<span class="co">#&gt;  [9] LC_ADDRESS=C               LC_TELEPHONE=C            </span>
<span class="co">#&gt; [11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       </span>
<span class="co">#&gt; </span>
<span class="co">#&gt; attached base packages:</span>
<span class="co">#&gt; [1] stats4    stats     graphics  grDevices utils     datasets  methods  </span>
<span class="co">#&gt; [8] base     </span>
<span class="co">#&gt; </span>
<span class="co">#&gt; other attached packages:</span>
<span class="co">#&gt;  [1] microbiomeDataSets_1.1.2       MultiAssayExperiment_1.19.5   </span>
<span class="co">#&gt;  [3] TreeSummarizedExperiment_2.1.3 Biostrings_2.61.1             </span>
<span class="co">#&gt;  [5] XVector_0.33.0                 SingleCellExperiment_1.15.1   </span>
<span class="co">#&gt;  [7] SummarizedExperiment_1.23.1    Biobase_2.53.0                </span>
<span class="co">#&gt;  [9] GenomicRanges_1.45.0           GenomeInfoDb_1.29.3           </span>
<span class="co">#&gt; [11] IRanges_2.27.0                 S4Vectors_0.31.0              </span>
<span class="co">#&gt; [13] BiocGenerics_0.39.1            MatrixGenerics_1.5.2          </span>
<span class="co">#&gt; [15] matrixStats_0.60.0             BiocStyle_2.21.3              </span>
<span class="co">#&gt; </span>
<span class="co">#&gt; loaded via a namespace (and not attached):</span>
<span class="co">#&gt;  [1] nlme_3.1-152                  bitops_1.0-7                 </span>
<span class="co">#&gt;  [3] fs_1.5.0                      bit64_4.0.5                  </span>
<span class="co">#&gt;  [5] filelock_1.0.2                httr_1.4.2                   </span>
<span class="co">#&gt;  [7] rprojroot_2.0.2               tools_4.1.0                  </span>
<span class="co">#&gt;  [9] bslib_0.2.5.1                 utf8_1.2.2                   </span>
<span class="co">#&gt; [11] R6_2.5.0                      DBI_1.1.1                    </span>
<span class="co">#&gt; [13] lazyeval_0.2.2                withr_2.4.2                  </span>
<span class="co">#&gt; [15] tidyselect_1.1.1              bit_4.0.4                    </span>
<span class="co">#&gt; [17] curl_4.3.2                    compiler_4.1.0               </span>
<span class="co">#&gt; [19] textshaping_0.3.5             desc_1.3.0                   </span>
<span class="co">#&gt; [21] DelayedArray_0.19.1           bookdown_0.22                </span>
<span class="co">#&gt; [23] sass_0.4.0                    rappdirs_0.3.3               </span>
<span class="co">#&gt; [25] pkgdown_1.6.1                 systemfonts_1.0.2            </span>
<span class="co">#&gt; [27] stringr_1.4.0                 digest_0.6.27                </span>
<span class="co">#&gt; [29] rmarkdown_2.9                 pkgconfig_2.0.3              </span>
<span class="co">#&gt; [31] htmltools_0.5.1.1             dbplyr_2.1.1                 </span>
<span class="co">#&gt; [33] fastmap_1.1.0                 rlang_0.4.11                 </span>
<span class="co">#&gt; [35] RSQLite_2.2.7                 shiny_1.6.0                  </span>
<span class="co">#&gt; [37] jquerylib_0.1.4               generics_0.1.0               </span>
<span class="co">#&gt; [39] jsonlite_1.7.2                BiocParallel_1.27.2          </span>
<span class="co">#&gt; [41] dplyr_1.0.7                   RCurl_1.98-1.3               </span>
<span class="co">#&gt; [43] magrittr_2.0.1                GenomeInfoDbData_1.2.6       </span>
<span class="co">#&gt; [45] Matrix_1.3-4                  Rcpp_1.0.7                   </span>
<span class="co">#&gt; [47] fansi_0.5.0                   ape_5.5                      </span>
<span class="co">#&gt; [49] lifecycle_1.0.0               stringi_1.7.3                </span>
<span class="co">#&gt; [51] yaml_2.2.1                    zlibbioc_1.39.0              </span>
<span class="co">#&gt; [53] BiocFileCache_2.1.1           AnnotationHub_3.1.4          </span>
<span class="co">#&gt; [55] grid_4.1.0                    blob_1.2.2                   </span>
<span class="co">#&gt; [57] promises_1.2.0.1              parallel_4.1.0               </span>
<span class="co">#&gt; [59] ExperimentHub_2.1.4           crayon_1.4.1                 </span>
<span class="co">#&gt; [61] lattice_0.20-44               KEGGREST_1.33.0              </span>
<span class="co">#&gt; [63] knitr_1.33                    pillar_1.6.2                 </span>
<span class="co">#&gt; [65] glue_1.4.2                    BiocVersion_3.14.0           </span>
<span class="co">#&gt; [67] evaluate_0.14                 BiocManager_1.30.16          </span>
<span class="co">#&gt; [69] png_0.1-7                     httpuv_1.6.1                 </span>
<span class="co">#&gt; [71] vctrs_0.3.8                   treeio_1.17.2                </span>
<span class="co">#&gt; [73] purrr_0.3.4                   assertthat_0.2.1             </span>
<span class="co">#&gt; [75] cachem_1.0.5                  xfun_0.24                    </span>
<span class="co">#&gt; [77] mime_0.11                     xtable_1.8-4                 </span>
<span class="co">#&gt; [79] tidytree_0.3.4                later_1.2.0                  </span>
<span class="co">#&gt; [81] ragg_1.1.3                    tibble_3.1.3                 </span>
<span class="co">#&gt; [83] AnnotationDbi_1.55.1          memoise_2.0.0                </span>
<span class="co">#&gt; [85] ellipsis_0.3.2                interactiveDisplayBase_1.31.2</span></code></pre></div>
</div>
</div>
</div>
  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">

        <nav id="toc" data-toggle="toc"><h2 data-toc-skip>Contents</h2>
    </nav>
</div>

</div>



      <footer><div class="copyright">
  <p>Developed by Leo Lahti, Felix G.M. Ernst, Sudarshan Shetty.</p>
</div>

<div class="pkgdown">
  <p>Site built with <a href="https://pkgdown.r-lib.org/">pkgdown</a> 1.6.1.</p>
</div>

      </footer>
</div>

  


  </body>
</html>
